### ISSUE TITLE
Music playing can work!

### ISSUE THREAD

**Issue by natis97** [openairplay/airplay2-receiver#26]
### Here are some great and positive views on this project and your work: 

So I've been following this topic for a long long time, I quite never managed to make anything turn my iMac into a legitimate AirPlay 2 device - however, this Project kept me interested in it. Many things & tools out there are told to deliver but honestly no one does ..... but **today a strange combo did it and enabled an outstanding sound-sync** / seamless streaming music & video on the Mac (... simultaneously with 2 HomePods + AirPort Express) 😍😍

I don't see any reason why such a good working receiver needs to hide behind real devices if the outcome is equally well. I'm not sure if you removed Sync on purpose because it is kinda experimental or just didn't get to that; however it is working perfectly and very impressive.

Great job and patience, I just got lucky and it turned out working, now I can finally check it off my list 😎 If this is interesting for others I'll sure share it with you guys but right now I'm just cheering

**Comment by tkrtz** [comment]
Hey, can you share your strange combo? I too am following this development for quite some time, but I didn't get it too work yet.
Thanks in advance!

**Comment by natis97** [comment]
> Hey, can you share your strange combo? I too am following this development for quite some time, but I didn't get it too work yet.
> Thanks in advance!

As promised find it here: [https://github.com/natis97/airplay2-receiver-sound ](https://github.com/natis97/airplay2-receiver-sound )

(I won't work on that actively and forked minor changes to enable sound, hope it enjoys / encourages you)

**Comment by systemcrash** [comment]
Can you summarize your changes? Are they effectively just this?:
```
                              {   'audioType': 'media',
                                  'inputLatencyMicros': 0,
                                  'outputLatencyMicros': 500000,
                                  'type': 103}
```
